get_matching_values/getmatchingvalue.js

```diff
@@ -0,0 +1,25 @@
+// receive 2 array. 
+// return matching value. if no match return null
```

Comment:
I think you were suppose to return some matching value (first is fine) and not all of them; otherwise, the question would have probably had said return an empty list and not null.